runelite-client/src/main/java/net/runelite/client/plugins/inventorytagger/InventoryTaggerConfig.java

```diff
@@ -0,0 +1,94 @@
+package net.runelite.client.plugins.inventorytagger;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+import java.awt.Color;
+
+@ConfigGroup(
+		keyName = "inventoryitemtagging",
+		name = "Inventory Item Tagging",
+		description = "Configuration for the Inventory Item Tagging"
```

Comment:
Wrong indentation, change continuation indent from 8 to 4 in idea (spans entire file)